egs/avspeech/local/predict.py

```diff
@@ -0,0 +1,163 @@
+#TODO: make it compatible with current changes
```

Comment:
Still relevant?

Reply:
I removed it completely since it is not being used... That's okay, right?

Reply:
Yes.